doc/08_dev_talks/paf24/student_roles24.md

```diff
@@ -0,0 +1,130 @@
+# Student Roles for Autonomous Vehicle Development Team
+
+## Role overview
+
+2-3 Students per Role
+
+- **Systems Engineer**
```

Comment:
Verantwortlichkeit für Abhängigkeiten von (neuen) Methoden und Modulen, insbesondere auf dem Zeitstrahl. Beispiel: Es ergibt keinen Sinn beim Decision Making viel Aufwand in die Verwendung von Stoplinien zu investieren, wenn klar ist, dass diese frühestens zum Projektende sensiert werden können.